include/matx/transforms/lu/lu_cuda.h

```diff
@@ -170,9 +170,15 @@ class matxDnLUCUDAPlan_t : matxDnCUDASolver_t {
           this->d_info + i);
 
       MATX_ASSERT(ret == CUSOLVER_STATUS_SUCCESS, matxSolverError);
```

Comment:
Can you change these to use `MATX_ASSERT_STR_EXP`? This will tell the user the cuSolver error code so it might help debug